qiskit/transpiler/passmanager.py

```diff
@@ -26,18 +28,27 @@
 
 
 class PassManager:
-    """A PassManager schedules the passes"""
+    """Manager for a set of Passes and their scheduling during transpilation."""
 
-    def __init__(self, passes=None, max_iteration=1000, callback=None):
-        """Initialize an empty PassManager object (with no passes scheduled).
+    def __init__(
+            self,
+            passes: Union[BasePass, List[BasePass]] = None,
+            max_iteration: int = 1000,
+            callback: Callable = None
```

Comment:
Spacing:
```
    def __init__(self,
                 passes: Union[BasePass, List[BasePass]] = None,
                 max_iteration: int = 1000,
                 callback: Callable = None):
```

Reply:
This applies elsewhere, too. If you really prefer this style, I believe you'd have to un-indent by 4 spaces

Reply:
Hmm on this one I was following mostly what we applied in the provider since we introduced typing (and argument lines started to become really long) - it took me a bit to get used to it, but found out that it does help readability - one argument per line, enough space for the complex types and the return type, and overall more "everything always in the same place" mood (at the expense of ~2 extra lines).

I can change the style if preferred - in the current version, the argument lines do need 4+4 spaces (["to distinguish arguments from the rest"](https://www.python.org/dev/peps/pep-0008/#indentation)), though.